phpunit/html/wp-html-tag-processor-standalone-test.php

```diff
@@ -294,7 +314,7 @@ public function test_set_attribute_prevents_xss( $attribute_value ) {
 		 * over the content and because we want to look at the raw values.
 		 */
 		$match = null;
-		preg_match( '~^<div test=(.*)></div>$~', (string) $p, $match );
+		preg_match( '~^<div test=(.*)></div>$~', $p->get_updated_html(), $match );
```

Comment:
why make this change here?

Reply:
Mostly because `(string) $p` is not readable, imho, and can be a bit confusing in PHP land.